### ISSUE TITLE
Move tutorial out of JuliaHEP-2023 repo

### ISSUE THREAD

**Issue by graeme-a-stewart** [JuliaHEP/JuliaHEP-2023#37]
Although the tutorial was given at the workshop it clearly lives on independently. So it should be in a different repo.

Either we move it to a new repo, or we rename this one to emphasise the tutorial material (other than a few PDFs, it is the real content that's here).

**Comment by Moelf** [comment]
We can just rename the repo

**Comment by graeme-a-stewart** [comment]
I tend to agree - rename is easier. Will that affect the deployment onto juliahep.org @Moelf ?

**Comment by Moelf** [comment]
I think we just gotta update url everything and it will be fine -- the github page contains a slug for the repo name

**Comment by Moelf** [comment]
as a result, link at https://hsf-training.org/training-center/ is broken

**Comment by szabo137** [comment]
Fair enough, who is able to fix that? @graeme-a-stewart ?

**Comment by Moelf** [comment]
https://github.com/hsf-training/training-center/blob/076ad0329be74258cfda9054bb4b4f9d14cac416/data/data.yaml#L121-L125
